models/users.js

```diff
@@ -0,0 +1,27 @@
+class Users {
+  constructor() {
+    this.list = [];
+  }
+
+  add(fakeName,realName) {
+    var new_user = {
+      fakeName,
+      realName,
+      id: this.generateID()
+    }
+    this.list.push(new_user);
+  }
+
```

Comment:
maybe just preference, but this way of generating ids seems a bit convoluted. 
Perhaps using a class variable to store id, and then incrementing that whenever you add a user.